.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "vcpkg"]
```

Comment:
Can this start with our license banner as a comment? For example, `.gitattributes` does.